deploy/helm/templates/statefulset.yaml

```diff
@@ -74,6 +74,11 @@ spec:
               containerPort: 662
               protocol: UDP
           securityContext:
+            {{- range $key, $value := .Values.extraArgs }}
+            {{- if and (eq $key "enable-xfs-quota") $value }}
```

Comment:
@mersl - have you tested this by passing the values for `extraArgs.enable-xfs-quota` with `true` and `false`.

Also, as this PR contains the changes from #38, can #38 be closed.

Reply:
The chart version needs to be bumped up.

Reply:
TL;DR
Yes - I checked setting enable-xfs-quota=false

I was unsure, if it's with me to bump chart's version. Happy to do so.


long:

the if clause works like:

($value == True) AND  ( $key eq "enable-xfs-quota)

as $value is already a bool, we don't need to compare if it's True. So as soon as enable-xfs-quota is set to false, the 'and's result is false, too.